aws/validators.go

```diff
@@ -1395,6 +1395,43 @@ func validateAwsKmsName(v interface{}, k string) (ws []string, es []error) {
 	return
 }
 
+func validateAwsKmsGrantOperation(v interface{}, k string) (ws []string, es []error) {
+	value := v.(string)
+
+	validTypes := map[string]bool{
```

Comment:
This could be a `map[string]struct{}` with each value set to `struct{}{}`, just to denote that the value of the map has no meaning.

Reply:
Why is this better than map[string]bool? Just to denote the value of the map has no meaning?